src/docs/development/add-to-app/android/project-setup.md

```diff
@@ -4,4 +4,226 @@ short-title: Integrate Flutter
 description: Learn how to integrate a Flutter module into your existing Android project.
 ---
 
-TODO(blasten): document.
\ No newline at end of file
+Flutter can be embedded into your existing Android application in a piecemeal
+fashion as a source code sub-project or as AARs.
+
+The integration flow can be done via the Android Studio IDE with the
+[Flutter plugin](https://plugins.jetbrains.com/plugin/9212-flutter) or manually.
+
+{{site.alert.warning}}
+Your existing Android app may currently support architectures such as `mips`
```

Comment:
Maybe, it doesn't need to be highlighted as a warning, but I think it's worth mentioning this limitation on this section.

Reply:
ya, not ideal but since every header below this point branches, this warning may be skipped over. i.e. if someone only uses Android Studio, they'll stop reading after the next heading and miss any warnings after.